modules/distribution/src/sso/views/register_new_user.jag

```diff
@@ -0,0 +1,200 @@
+                            <div class="padding-double">
+                                <div id="regFormError" class="alert alert-error" style="display:none"></div>
+                                <div id="regFormSuc" class="alert alert-success" style="display:none"></div>
+
+                                    <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12 form-group">
+                                        <label>Username</label>
+                                        <input id="reg-username" name="reg-username" type="text" class="form-control required usrName usrNameLength">
+                                    </div>
+                                    <div class="col-xs-12 col-sm-12 col-md-6 col-lg-6 form-group">
+                                        <label>Password</label>
+                                        <input id="reg-password" name="reg-password" type="password" class="form-control required passwordLength">
+                                    </div>
+                                    <div class="col-xs-12 col-sm-12 col-md-6 col-lg-6 form-group">
+                                        <label>Confirm password</label>
+                                        <input id="reg-password2" name="password2" type="password" class="form-control required" equalTo="#reg-password">
+                                    </div>
+
+                                <%
+                                    var claimsModule = require('account-management');
+                                    var claims = claimsModule.getDefaultClaims();
+                                    var renderedClaims = {};
+
+                                    var claimUriOverrides = {
+                                        USERNAME: {
+                                            render: false // because this is already rendered by default
+                                        },
+                                        PASSWORD: {
+                                            render: false // because this is already rendered by default
+                                        },
+                                        EMAILADDRESS: {
+                                            render: true,
+                                            customClass: 'email'
+                                        }
+                                    };
+
+                                    for (var claimIndex = 0; claimIndex < claims.length; claimIndex++) {
+                                        var override = claimUriOverrides[claims[claimIndex].claimUri.toUpperCase()];
+                                        // render claim if it's required and not overriden to --> render:false
+                                        if (claims[claimIndex].isRequired && (!override || override.render !== false)) {
+                                            var formElementName = 'reg-' + claims[claimIndex].displayTag.toLowerCase().replace(' ', '-');
```

Comment:
Be consistent on using quotes for string definitions. use double quotes as the standard.